src/server/journal/journal_slice.cc

```diff
@@ -51,6 +55,7 @@ void JournalSlice::Init(unsigned index) {
 
   slice_index_ = index;
   ring_buffer_.emplace(128);  // TODO: to make it configurable
+  pings_fb_ = MakeFiber(&JournalSlice::PingsLoop, this, 3000ms);
```

Comment:
So if we dont have replicas we still start this fiber?

Reply:
Yes, I didn't think it makes that much of a difference